contributing.md

```diff
@@ -17,7 +17,14 @@ feel free to propose changes to this document in a pull request.
 
 ## Translating
 
-👉 **Do not translate the content directly in this repository!**
+Here are some guidelines to keep in mind as a translator:
+
+- All translation happens on [Crowdin](https://crowdin.com/project/electron). **Do not open pull requests that translate the content in this repository!**
+- Do not translate JavaScript keywords like `String`, `Event`, `Array`, `Class`, etc.
+- Do not translate Electron classes, method names, event names, etc.
```

Comment:
_(trivial)_ If using `do not` instead of `don't` here as a style rule, remove contractions below e.g. `you've` and `you're`. Otherwise, use `don't` here :smile: